src/gsi/convb_ps.f90

```diff
@@ -57,6 +57,7 @@ subroutine convb_ps_read(mype)
 !                          the hardwired value in the calculation of table array index.
 !                          ld=300 is sufficient for current conventional
 !                          observing systems.
+!   2025-04-17  collard -- Increase ld to 500.
```

Comment:
It's fine to keep this but since we now use version control we no longer need to maintain a change log in the source code.